build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/release/ChangelogEntry.java

```diff
@@ -215,6 +215,7 @@ public String toString() {
         private String details;
         private String impact;
         private boolean notable;
+        private boolean essSettingChange;
```

Comment:
Where is this actually used? Only place I can find is in test code.

Reply:
It's used in the `breaking-changes-area.asciidoc` template.

Reply:
I don't see where we ever actually _set_ this value though. Am I missing something I guess folks are expected to explicitly add this to the changelog entry file?

Reply:
Yes, exactly. Where we make a change / add / remove a setting with an impact to ESS, we can set this new field so that this setting has the appropriate icon in the docs. That's all.